templates/bot/js/notification-restify/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "echobot",
```

```suggestion
  "name": "notification-restify-bot",
```
to be consistent with other templates

---

fixed